api/v1alpha1/flowcollector_types.go

```diff
@@ -55,6 +55,7 @@ type FlowCollectorSpec struct {
 	Processor FlowCollectorFLP `json:"processor,omitempty"`
 
 	// loki, the flow store, client settings.
+	// +k8s:conversion-gen=false
```

Comment:
I don't think it was the intent, but that said, we should think about removing alpha1 ... Since it's alpha we don't even need to go through a deprecation notice

Reply:
Removed in https://github.com/netobserv/network-observability-operator/pull/329/commits/656cb0fb4fd5982bad0d6f0110a2eb1bb4f2e898

What's the best practice here ? Removing `v1alpha1` while introducing `v1beta2` or wait for next release ?

Reply:
deprecated in https://github.com/netobserv/network-observability-operator/pull/329/commits/282b28ff4a20fdc7242a1903978b0cb098100565